### DISCUSSION TITLE
Hide archived entries in relation selection if Archive filter is on

### DISCUSSION THREAD

**Discussion by darioguarascio** [directus/directus#8896] [Feature Requests]
I was playing around with the archive feature, it works great by not showing archived entries by default in collection data view.
but on other tables with relations to the archived-enabled collection, when selecting entries, i get the whole list, including archived ones.
Wouldn't it make sense to have the archive filter affect every place the collection shows up ?

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
